server/gamenode/Lobby.ts

```diff
@@ -23,23 +25,49 @@ export class Lobby {
     // switch partic
     private users: LobbyUser[] = [];
     private tokens: { battleDroid: any; cloneTrooper: any; experience: any; shield: any };
+    private gameChat: GameChat;
+
     public constructor() {
         this._id = uuid();
+        this.gameChat = new GameChat();
     }
 
     public get id(): string {
         return this._id;
     }
 
-    public createLobbyUser(user, deck = null): void {
+    public getLobbyState(): any {
+        return {
+            id: this._id,
+            users: this.users.map((u) => ({
+                id: u.id,
+                username: u.username,
+                state: u.state,
+                ready: u.ready,
+                deck: u.deck?.data,
+                owner: u.owner
+            })),
+            gameChat: this.gameChat,
```

Comment:
Would it be useful to have like the id or username of the owner as top level property?